backends-common/elasticsearch-v7/src/main/java/org/apache/james/backends/es/v7/NodeMappingFactory.java

```diff
@@ -60,18 +59,18 @@ public static ReactorElasticSearchClient applyMapping(ReactorElasticSearchClient
         return client;
     }
 
-    // ElasticSearch 6.3.2 does not support field master_timeout that is set up by 6.4.3 REST client when relying on getMapping
-    @SuppressWarnings("deprecation")
     public static boolean mappingAlreadyExist(ReactorElasticSearchClient client, IndexName indexName) throws IOException {
```

Comment:
Where is this method used? Do we need it in production code?

Reply:
i don't think so. Only used in tests now. Should we remove this class along with all its tests?

Reply:
IMO yes